src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,78 @@
+import Text from "src/components/Text";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import { useState } from "react";
+
+const ProfileContent: React.FC<Record<string, never>> = () => {
+  const [name, setName] = useState<string>(undefined);
+  const [subject, setSubject] = useState<string>(undefined);
+  const [content, setContent] = useState<string>(undefined);
```

Comment:
I think its better to have string | undefined just to make its explicit that it can be undefined

Reply:
ok